pkg/app/two_pass_renderer.go

```diff
@@ -89,38 +90,56 @@ func (r *desiredStateLoader) twoPassRenderTemplateToYaml(inherited, overrode *en
 		return nil, err
 	}
 
-	if r.logger != nil {
-		r.logger.Debugf("first-pass uses: %v", initEnv)
-	}
+	var (
+		finalEnv *environment.Environment
+		vals     map[string]interface{}
+	)
 
-	renderedEnv, prestate := r.renderPrestate(initEnv, baseDir, filename, content)
+	if runtime.V1Mode {
+		var err error
```

Comment:
other code also output `second-pass rendering` or `fisrt-pass rendering` in v1 mode. I think that should disable the output. @mumoshu

Reply:
Good point! Let me fix it asap.

Reply:
See 4943483